lib/inferno/dsl/jwks.rb

```diff
@@ -0,0 +1,27 @@
+module Inferno
+  module DSL
+    class JWKS
+      class << self
+        def jwks_json
+          @jwks_json ||=
+            JSON.pretty_generate(
+              { keys: jwks.export[:keys].select { |key| key[:key_ops]&.include?('verify') } }
+            )
+        end
+
+        def default_jwks_path
+          @default_jwks_path ||= File.join(__dir__, 'jwks.json')
+        end
+
+        def jwks_path
+          @jwks_path ||=
+            ENV.fetch('CORE_JWKS_PATH', default_jwks_path)
+        end
+
+        def jwks(user_jwks: nil)
+          @jwks ||= JWT::JWK::Set.new(JSON.parse(user_jwks.present? ? user_jwks : File.read(jwks_path)))
```

Comment:
Is `user_jwks` for using a jwks supplied as an input? The memoization is going to set `@jwks` globally since this is a class method.

Reply:
Yes, `user_jwks` is for using a jwks input. Don't we want that? 
I'll remove the `@jwks`.